models/bin.js

```diff
@@ -26,9 +26,8 @@ async function binByUUID(id) {
 
 async function newBin() {
   try {
-    await client.query("INSERT INTO bins DEFAULT VALUES");
-    const newUUID = await client.query("SELECT uuid FROM bins ORDER BY id DESC LIMIT 1");
-    return newUUID.rows;
+    const newUUID = await client.query("INSERT INTO bins DEFAULT VALUES RETURNING id");
```

Comment:
I think we actually want to return `uuid` here (the long random generated string) not `id` which is just an auto incrementing integer.

Reply:
good catch, fixed!